spec/lib/journaled_spec.rb

```diff
@@ -49,4 +49,20 @@
       expect(described_class.actor_uri).to eq "my actor uri"
     end
   end
+
+  describe '.stream_name_for_app' do
+    before do
+      allow(ENV).to receive(:fetch).and_return("expected_stream_name")
+    end
+
+    it "is fetched from a prefixed ENV var if specified when app_name is unspecified" do
+      expect(described_class.stream_name_for_app(nil)).to eq("expected_stream_name")
+      expect(ENV).to have_received(:fetch).with("JOURNALED_STREAM_NAME")
+    end
+
+    it "is fetched from a prefixed ENV var if specified when app_name is specified" do
+      expect(described_class.stream_name_for_app("my_funky_app_name")).to eq("expected_stream_name")
+      expect(ENV).to have_received(:fetch).with("MY_FUNKY_APP_NAME_JOURNALED_STREAM_NAME")
```

Comment:
consistency nit: the other specs in below seem to use single quotes 😊

Reply:
Wow I wonder why rubocop didn't catch that! 🦅 👁

Reply:
Huh so Rubocop in this repo wasn't enabled to check for Strings :panic:

And since it was gonna change like half the repo lol I broke it out into a seperate PR, just for organization and later upstreaming easier, so here is a seperate PR for just the quotes, and once I merge that I will fix that quotes I added here too!
https://github.com/WellSheet/journaled/pull/3